src/utils/pgwire/src/pg_message.rs

```diff
@@ -51,6 +51,15 @@ pub enum FeMessage {
     ServerThrottle(ServerThrottleReason),
 }
 
+impl FeMessage {
+    pub fn get_sql(&self) -> Result<Option<&str>> {
```

Comment:
This is not good. We did extra redaction and truncation to SQL before recording it.

https://github.com/risingwavelabs/risingwave/blob/61722a70bd6ffa5839eb375503284d866785f2c8/src/utils/pgwire/src/pg_protocol.rs#L178-L193

Reply:
Let me add it.